### PR TITLE
Iteraciones en Requisitos Previos en base a Overview

### PR DESCRIPTION
**Pull request by abderossi**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/in-person-payments/qr-code/pre-requisites.es.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ Sabemos que algunos términos son nuevos. Antes de empezar, te los dejamos a man
...
 
-Para continuar, es necesario realizar los siguientes pasos:
+Ten en cuenta estos aspectos antes de empezar:
 
-### 1. Acceso a cuenta de Mercado Pago o Mercado Libre
+### 1. Accede a una cuenta
 
 Para poder comenzar la integración, es necesario **contar con una cuenta de Mercado Pago o Mercado Libre**. 
-Si aún no tienes una, puedes [crear una cuenta de Mercado Pago](https://www.mercadopago.com.ar) cuando quieras.
+Puedes [Ingresar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) a una cuenta de Mercado Pago o de Mercado Libre que ya exista o [Crear una cuenta nueva](https://www.mercadopago[FAKER][URL][DOMAIN]) en Mercado Pago.
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/pre-requisites.es.md:37]:
```suggestion

Puedes [Ingresar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/in-person-payments/qr-code/pre-requisites) a una cuenta de Mercado Pago o de Mercado Libre que ya exista o [Crear una cuenta nueva](https://www.mercadopago[FAKER][URL][DOMAIN]) en Mercado Pago.
```

File: guides/in-person-payments/qr-code/pre-requisites.pt.md [hunk 2]
```diff
@@ -29,11 +29,13 @@ Sabemos que alguns termos são novos. Antes de começar, os deixamos perto de su
...
-Para continuar, é necessário realizar o passos a seguir: 
+Observe estes pontos antes de começar: 
 
-### 1. Acesso à conta de Mercado Pago ou Mercado Livre
+### 1. Acesse uma conta
 
-Para poder começar a integração, é necessário **contar com uma conta de Mercado Pago ou Mercado Livre**. Se ainda não tiver uma, poderá [criar uma conta de Mercado Pago](https://www.mercadopago.com.br) quando quiser.
+Para poder começar a integração, é necessário **contar com uma conta de Mercado Pago ou Mercado Livre**. 
+
+Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/pre-requisites.pt.md:38]:
```suggestion

Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/in-person-payments/qr-code/pre-requisites) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```

File: guides/online-payments/checkout-pro/previous-requirements.pt.md [hunk 3]
```diff
@@ -15,13 +15,13 @@ Sabemos que alguns termos são novos. Antes de começar, os deixamos à mão.
...
+Observe estes pontos antes de começar a integração:
 
-### 1. Acesso à conta do Mercado Pago ou do Mercado Livre
+### 1. Acesse uma conta
 Para poder começar a integração, é necessário **contar com uma conta do Mercado Pago ou do Mercado Livre.**
-Caso você ainda não tenha uma, pode [criar uma conta do Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]) quando quiser.
+Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
 
-### 2.  Instalação do SDK do Mercado Pago
+### 2.  Instale do SDK do Mercado Pago
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/previous-requirements.pt.md:24]:
```suggestion
### 2. Instale do SDK do Mercado Pago
```

File: guides/online-payments/checkout-pro/previous-requirements.en.md [hunk 4]
```diff
@@ -15,13 +15,14 @@ We know some terms are new. Before getting started, we'll give you a hand.
...
+Keep these aspects in mind before you start integrating:
 
-### 1. Access to the Mercado Pago or Mercado Libre account
-In order to start the integration, it is necessary to **have a Mercado Pago or Mercado Libre account.**.
-If you don't have one yet, you can [create a Mercado Pago account](https://www.mercadopago[FAKER][URL][DOMAIN]) whenever you want.
+### 1. Access to an account
 
-### 2. Installations of Mercado Pago SDK
+In order to start the integration, it is necessary to **have a Mercado Pago or Mercado Libre account**.
+You can [Sign in](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) an existing Mercado Pago or Mercado Libre account or [Create a new account](https://www.mercadopago[FAKER][URL][DOMAIN]) at Mercado Pago.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/previous-requirements.en.md:23]:
```suggestion

You can [Sign in](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-pro/previous-requirements) an existing Mercado Pago or Mercado Libre account or [Create a new account](https://www.mercadopago[FAKER][URL][DOMAIN]) at Mercado Pago.
```

File: guides/online-payments/checkout-pro/previous-requirements.es.md [hunk 5]
```diff
@@ -15,14 +15,14 @@ Sabemos que algunos términos son nuevos. Antes de empezar, te los dejamos a man
...
 ## Requisitos previos
 
-Para continuar, verifica los requisitos previos necesarios:
+Ten en cuenta estos aspectos antes de empezar a integrar: 
 
-### 1. Acceso a cuenta de Mercado Pago o Mercado Libre
+### 1. Accede a una cuenta
 Para poder comenzar la integración, es necesario **contar con una cuenta de Mercado Pago o Mercado Libre**.
-Si aún no tienes una, puedes [crear una cuenta de Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]) cuando quieras.
+Puedes [Ingresar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) a una cuenta de Mercado Pago o de Mercado Libre que ya exista o [Crear una cuenta nueva](https://www.mercadopago[FAKER][URL][DOMAIN]) en Mercado Pago.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/previous-requirements.es.md:22]:
```suggestion

Puedes [Ingresar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-pro/previous-requirements) a una cuenta de Mercado Pago o de Mercado Libre que ya exista o [Crear una cuenta nueva](https://www.mercadopago[FAKER][URL][DOMAIN]) en Mercado Pago.
```

File: guides/online-payments/checkout-pro/previous-requirements.pt.md [hunk 6]
```diff
@@ -15,13 +15,13 @@ Sabemos que alguns termos são novos. Antes de começar, os deixamos à mão.
...
 ## Pré-requisitos
 
-Para continuar, confira os pré-requisitos:
+Observe estes pontos antes de começar a integração:
 
-### 1. Acesso à conta do Mercado Pago ou do Mercado Livre
+### 1. Acesse uma conta
 Para poder começar a integração, é necessário **contar com uma conta do Mercado Pago ou do Mercado Livre.**
-Caso você ainda não tenha uma, pode [criar uma conta do Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]) quando quiser.
+Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/previous-requirements.pt.md:22]:
```suggestion

Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-pro/previous-requirements) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```

File: guides/online-payments/subscriptions/previous-requirements.en.md [hunk 7]
```diff
@@ -26,14 +26,14 @@ Please note that the minimum amount allowed to create a subscription is $ 10 and
 
 ## First steps
 
-### Access to the Mercado Pago or Mercado Libre account
-In order to start the integration, it is necessary to have a Mercado Pago or Mercado Libre account.
+### Access to an account
+In order to start the integration, it is necessary to **have a Mercado Pago or Mercado Libre account**.
 
-If you don't have one yet, you can [create a account](https://www.mercadopago[FAKER][URL][DOMAIN]) whenever you want.
+You can [Sign in](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) an existing Mercado Pago or Mercado Libre account [Create a new account](https://www.mercadopago[FAKER][URL][DOMAIN]) at Mercado Pago.
```
Comment (pamalv7) [inline guides/online-payments/subscriptions/previous-requirements.en.md:32]:
```suggestion
You can [Sign in](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/subscriptions/previous-requirements) an existing Mercado Pago or Mercado Libre account [Create a new account](https://www.mercadopago[FAKER][URL][DOMAIN]) at Mercado Pago.
```

File: guides/online-payments/subscriptions/previous-requirements.es.md [hunk 8]
```diff
@@ -26,14 +26,15 @@ Ten en cuenta que el monto mínimo permitido para crear una suscripción es de $
...
 ## Primeros pasos
 
-### Acceso a cuenta de Mercado Pago o Mercado Libre
-Para poder comenzar la integración, es necesario contar con una cuenta de Mercado Pago o Mercado Libre.
+### Accede a una cuenta
 
-Si aún no tienes una, puedes [crear una cuenta](https://www.mercadopago[FAKER][URL][DOMAIN]) cuando quieras.
+Para poder comenzar la integración, es necesario **contar con una cuenta de Mercado Pago o Mercado Libre**. 
+
+Puedes [Ingresar]([https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview](https://www.mercadolibre.com/jms/%5BFAKER%5D%5BGLOBALIZE%5D%5BSITE_ID%5D/lgz/login?platform_id=mp&go=https://www.mercadopago%5BFAKER%5D%5BURL%5D%5BDOMAIN%5D/developers/es/guides/overview)) a una cuenta de Mercado Pago o de Mercado Libre que ya exista o [Crear una cuenta nueva](https://www.mercadopago[FAKER][URL][DOMAIN]) en Mercado Pago.
```
Comment (pamalv7) [inline guides/online-payments/subscriptions/previous-requirements.es.md:33]:
```suggestion
Puedes [Ingresar]([https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview](https://www.mercadolibre.com/jms/%5BFAKER%5D%5BGLOBALIZE%5D%5BSITE_ID%5D/lgz/login?platform_id=mp&go=https://www.mercadopago%5BFAKER%5D%5BURL%5D%5BDOMAIN%5D/developers/es/guides/online-payments/subscriptions/previous-requirements) a una cuenta de Mercado Pago o de Mercado Libre que ya exista o [Crear una cuenta nueva](https://www.mercadopago[FAKER][URL][DOMAIN]) en Mercado Pago.
```

File: guides/online-payments/subscriptions/previous-requirements.pt.md [hunk 9]
```diff
@@ -26,14 +26,15 @@ Observe que o valor mínimo permitido para criar uma assinatura é de $ 10 e o m
...
 ## Primeiros passos
 
-### Acesso à conta Mercado Pago ou Mercado Livre
-Para poder começar a integração, é necessário ter uma conta Mercado Pago ou Mercado Livre.
+### Acesse uma conta
 
-Se você ainda não tem uma, pode [criar uma conta](https://www.mercadopago[FAKER][URL][DOMAIN]) quando quiser.
+Para poder começar a integração, é necessário **contar com uma conta do Mercado Pago ou do Mercado Livre**.
+
+Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/overview) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```
Comment (pamalv7) [inline guides/online-payments/subscriptions/previous-requirements.pt.md:33]:
```suggestion
Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/subscriptions/previous-requirements) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

File: guides/in-person-payments/qr-code/pre-requisites.pt.md [hunk 10]
```diff
@@ -29,11 +29,14 @@ Sabemos que alguns termos são novos. Antes de começar, os deixamos perto de su
...
-Para continuar, é necessário realizar o passos a seguir: 
+Observe estes pontos antes de começar: 
 
-### 1. Acesso à conta de Mercado Pago ou Mercado Livre
+### 1. Acesse uma conta
 
-Para poder começar a integração, é necessário **contar com uma conta de Mercado Pago ou Mercado Livre**. Se ainda não tiver uma, poderá [criar uma conta de Mercado Pago](https://www.mercadopago.com.br) quando quiser.
+Para poder começar a integração, é necessário **contar com uma conta de Mercado Pago ou Mercado Livre**. 
+
+
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/pre-requisites.pt.md:38]:
```suggestion
```

**Review by pamalv7 — APPROVED** [review]

File: guides/in-person-payments/qr-code/pre-requisites.pt.md [hunk 11]
```diff
@@ -29,11 +29,13 @@ Sabemos que alguns termos são novos. Antes de começar, os deixamos perto de su
...
-Para continuar, é necessário realizar o passos a seguir: 
+Observe estes pontos antes de começar: 
 
-### 1. Acesso à conta de Mercado Pago ou Mercado Livre
+### 1. Acesse uma conta
 
-Para poder começar a integração, é necessário **contar com uma conta de Mercado Pago ou Mercado Livre**. Se ainda não tiver uma, poderá [criar uma conta de Mercado Pago](https://www.mercadopago.com.br) quando quiser.
+Para poder começar a integração, é necessário **contar com uma conta de Mercado Pago ou Mercado Livre**. 
+
+Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/in-person-payments/qr-code/pre-requisites) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```
Comment (lamaralmp) [inline guides/in-person-payments/qr-code/pre-requisites.pt.md:38]:
```suggestion
Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/in-person-payments/qr-code/pre-requisites) em uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```

File: guides/online-payments/checkout-pro/previous-requirements.pt.md [hunk 12]
```diff
@@ -15,13 +15,14 @@ Sabemos que alguns termos são novos. Antes de começar, os deixamos à mão.
...
-Para continuar, confira os pré-requisitos:
+Observe estes pontos antes de começar a integração:
 
-### 1. Acesso à conta do Mercado Pago ou do Mercado Livre
+### 1. Acesse uma conta
 Para poder começar a integração, é necessário **contar com uma conta do Mercado Pago ou do Mercado Livre.**
-Caso você ainda não tenha uma, pode [criar uma conta do Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]) quando quiser.
 
-### 2.  Instalação do SDK do Mercado Pago
+Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-pro/previous-requirements) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```
Comment (lamaralmp) [inline guides/online-payments/checkout-pro/previous-requirements.pt.md:23]:
```suggestion
Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-pro/previous-requirements) em uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```

File: guides/online-payments/subscriptions/previous-requirements.pt.md [hunk 13]
```diff
@@ -26,14 +26,15 @@ Observe que o valor mínimo permitido para criar uma assinatura é de $ 10 e o m
...
 ## Primeiros passos
 
-### Acesso à conta Mercado Pago ou Mercado Livre
-Para poder começar a integração, é necessário ter uma conta Mercado Pago ou Mercado Livre.
+### Acesse uma conta
 
-Se você ainda não tem uma, pode [criar uma conta](https://www.mercadopago[FAKER][URL][DOMAIN]) quando quiser.
+Para poder começar a integração, é necessário **contar com uma conta do Mercado Pago ou do Mercado Livre**.
+
+Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/subscriptions/previous-requirements) uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```
Comment (lamaralmp) [inline guides/online-payments/subscriptions/previous-requirements.pt.md:33]:
```suggestion
Você pode [Entrar](https://www.mercadolibre.com/jms/[FAKER][GLOBALIZE][SITE_ID]/lgz/login?platform_id=mp&go=https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/subscriptions/previous-requirements) em uma conta existente no Mercado Pago ou no Mercado Livre ou [Criar uma nova conta](https://www.mercadopago[FAKER][URL][DOMAIN]) no Mercado Pago.
```

File: guides/online-payments/checkout-pro/previous-requirements.pt.md [hunk 14]
```diff
@@ -68,6 +69,13 @@ PM> Install-Package mercadopago-sdk -Version 1.0.57
 ```
 ]]]
 
+**### 3. Obtenha suas credenciais**
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/previous-requirements.pt.md:72]:
```suggestion
### 3. Obtenha suas credenciais
```
¿Por qué le sumamos negritas? Por lo que recuerdo no funcionan con la nueva lib en los títulos.

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by lamaralmp — APPROVED** [review]

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**